scout/constants/case_tags.py

```diff
@@ -71,6 +71,7 @@
     "cancel_sanger": "cancelled sanger order for",
     "archive": "archived",
     "open_research": "opened research mode for",
+    "close_research": "canceled research mode for",
```

Comment:
```suggestion
    "reset_research": "canceled research mode request for",
```

Reply:
It's not really requesting to close research mode. Which could totally be a thing, but for another issue! Reset is good.